### PR TITLE
Add release notes for v0.1.8

### PR DESCRIPTION
**Pull request by Gedochao**

### CODE REVIEW

**Review by romanowski — APPROVED** [review]

File: .github/release/release_notes.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+Hello world
+```
+
+Changed in [#1095](https://github.com/VirtusLab/scala-cli/pull/1095) by @Gedochao
+
+## Important bugfixes
+
+### Enable downloading Scala versions outside the Coursier cache
+
+Scala CLI is now able to download the newest Scala versions, even if they're not yet available in the Coursier cache. In
```
Comment (alexarchambault) [inline .github/release/release_notes.md]: "even if they're not yet available in the Coursier cache" doesn't sound right to me (we used to have a problem if they're not in the Scala version listing in the cache). I'd say it like this instead:

Scala CLI uses version listings from Maven Central to check if a Scala version is valid. When new Scala versions are released, users could sometimes have stale version listings in their Coursier cache for a short period of time (the Coursier cache TTL, which is 24 hours by default). This prevented these users to use new Scala versions during that time.
To work around that, Scala CLI now tries to re-download version listings when they don't have the requested Scala version.

File: .github/release/release_notes.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+case of a cache miss, it force-retries and fetches the version, if at all possible.
+This addresses [#1090](https://github.com/VirtusLab/scala-cli/issues/1090)
+
+Fixed in [#1096](https://github.com/VirtusLab/scala-cli/pull/1096) by @lwronski
+
+### Bloop now uses `JAVA_HOME` by default
+
+Bloop should now pick up on the JDK available in `JAVA_HOME`. It was formerly necessary to pass `--bloop-jvm system`
```
Comment (alexarchambault) [inline .github/release/release_notes.md]: ?
```suggestion
Bloop should now pick up the JDK available in `JAVA_HOME`. It was formerly necessary to pass `--bloop-jvm system`
```

File: .github/release/release_notes.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+### Fixes
+
+* Add suffix to project name which contains virtual files by @lwronski
+  in [#1070](https://github.com/VirtusLab/scala-cli/pull/1070)
+
+### Build and internal changes
+
+* Update scala-cli.sh launcher for 0.1.7 by @github-actions in [#1076](https://github.com/VirtusLab/scala-cli/pull/1076)
+* Add config command by @alexarchambault in [#1056](https://github.com/VirtusLab/scala-cli/pull/1056)
```
Comment (alexarchambault) [inline .github/release/release_notes.md]: Do we need this line, as there's a section about it above?
```suggestion
```

Comment (Gedochao) [reply .github/release/release_notes.md]: Ah yeah, indeed we don't

File: .github/release/release_notes.md (hunk 1 above)
Comment (Gedochao) [reply .github/release/release_notes.md]: Applied

File: .github/release/release_notes.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+You can run any one of them by passing option --main-class, i.e. --main-class scripts.AnotherScript_sc
+All available main classes can always be listed by passing option --list-main-classes
+Hello world
+```
+
+Changed in [#1095](https://github.com/VirtusLab/scala-cli/pull/1095) by @Gedochao
+
+## Important bugfixes
+
+### Enable downloading Scala versions outside the Coursier cache
```
Comment (alexarchambault) [inline .github/release/release_notes.md]: Maybe we can change this too:
```suggestion
### Support latest Scala versions despite stale Scala version listings in cache
```

Comment (Gedochao) [reply .github/release/release_notes.md]: applied

File: .github/release/release_notes.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+You can run any one of them by passing option --main-class, i.e. --main-class scripts.AnotherScript_sc
+All available main classes can always be listed by passing option --list-main-classes
+Hello world
+```
+
+Changed in [#1095](https://github.com/VirtusLab/scala-cli/pull/1095) by @Gedochao
+
+## Important bugfixes
+
+### Support latest Scala versions despite stale Scala version listings in cache
```
Comment (alexarchambault) [inline .github/release/release_notes.md]: Last change 😅
```suggestion
### Accept latest Scala versions despite stale Scala version listings in cache
```

**Review by alexarchambault — APPROVED** [review]

**Review by lwronski — APPROVED** [review]

**Pull request merged**